Taarafo.Portal.Web/Program.cs

```diff
@@ -17,9 +18,19 @@ public static void Main(string[] args)
 
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
-                .ConfigureWebHostDefaults(webBuilder =>
+            .ConfigureAppConfiguration((hostingContext, config) =>
+            {
+                var environment = hostingContext.HostingEnvironment;
+                config.AddJsonFile("appsettings.json", optional: true, reloadOnChange: true);
+
+                if (!string.IsNullOrEmpty(environment.EnvironmentName))
                 {
-                    webBuilder.UseStartup<Startup>();
-                });
+                    config.AddJsonFile($"appsettings.{environment}.json", optional: true, reloadOnChange: true);
+                };
+            })
```

Comment:
@cjdutoit one more thing here. I think ASP.NET does that outta box when you setup the environment in the `launchSettings.json` under this code:
```json
 "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }

```
If I'm not mistaken this is what you're trying to accomplish here right?

Reply:
@hassanhabib did not realize that is now out of the box.  Will remove this as well.